ggml/src/ggml-sycl/ggml-sycl.cpp

```diff
@@ -3031,19 +3049,54 @@ static bool ggml_sycl_supports_dmmv(enum ggml_type type) {
     }
 }
 
+// Helper functions to unify device memory allocation for both async and sync paths
+static inline void * sycl_malloc_opt_async(dpct::queue_ptr  stream,
```

Comment:
```suggestion
static inline void * sycl_ext_malloc(dpct::queue_ptr  stream,
```

Reply:
Renamed to `sycl_ext_malloc_device` to align with SYCL style naming.